devicon.json

```diff
@@ -4425,6 +4425,33 @@
         "color": "#6ba43a",
         "aliases": []
     },
+    {
+        "name": "opencl",
+        "altnames": [],
+        "tags": [
+            "framework",
+            "language",
+            "heterogeneous-computing",
+            "cpp",
+            "api",
+            "khronos"
```

Comment:
You may want to add `programming` here as well. 🙂
```suggestion
            "khronos",
            "programming"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
okay